extensions-core/google-extensions/src/test/java/org/apache/druid/data/input/google/GoogleCloudStorageInputSourceTest.java

```diff
@@ -169,14 +175,18 @@ public void testWithPrefixesSplit() throws IOException
   public void testReader() throws IOException
   {
     EasyMock.reset(STORAGE);
+    EasyMock.reset(CONFIG);
     addExpectedPrefixObjects(PREFIXES.get(0), ImmutableList.of(EXPECTED_URIS.get(0)));
     addExpectedGetObjectMock(EXPECTED_URIS.get(0));
     addExpectedPrefixObjects(PREFIXES.get(1), ImmutableList.of(EXPECTED_URIS.get(1)));
     addExpectedGetObjectMock(EXPECTED_URIS.get(1));
+    EasyMock.expect(CONFIG.getMaxListingLength()).andReturn(EXPECTED_MAX_LISTING_LENGTH);
     EasyMock.replay(STORAGE);
+    EasyMock.replay(CONFIG);
```

Comment:
nit: looks like this is repeated in multiple tests, maybe move to a helper function?